_posts/2023-03-02-edf-uk-energy-industry-interactions-eda.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Managing complex UK Energy Interactions with EDA"
+summary: "How EDF Energy is using Event Driven Architecture to manage the complexity of UK Energy Inetractions"
+date: 2023-03-02T19:15:00
+meetup: https://www.meetup.com/Async-Web-Tech-Meetup/
+speakers:
+- name: Kev Baldwyn
+  link: https://www.linkedin.com/in/kevbaldwyn/
+sponsors:
+image:
+  url:   https://live.staticflickr.com/65535/50601725777_1a756e08d0_b_d.jpg
+  title: "'evening energy' by Joachim Aspenlaub Blattboldt"
+  link:  https://www.flickr.com/photos/blattboldt/50601725777
+tags:
+- EDF
+- EDA
+- AWS
+- Event Driven Architecture
+venue:
+  name: Spaces
+  link: https://www.spacesworks.com/brighton/trafalgar-place/
+  location: https://g.page/Mocatta-House-4437?share
+  address: Mocatta House, Trafalgar Pl, BN1 4DU, Brighton
+  latlong: 50.8294399,-0.1443907
+layout: event.html
+collection: events
+draft: true # Delete this line to publish
+---
+
+The UK energy industry is made up of lots of parties that need to communicate information with each other. Historically the methods used have been largely file based, with EDF needing to share the data internally between many different systems for many different needs. 
+
+De-regulation, a more competitive energy market, a rising cost to serve energy and changes to industry processes has meant EDF needs to find ways to drive it's cost to serve down and stay competitive.
+
+Event Driven Architecture is increasingly becoming the tool that EDF is choosing to manage the complexity of these energy interactions and solve these problems.
+
+Kevin Baldwyn is a lead engineer at EDF energy and in this talk he will explain how EDF is using EDA to manage the complexity of UK Energy Interactions and describe some of the architecture, patterns, lessons learned, and why you may want to start using EDA in your applications.
```

Comment:
Please don't commit this yet. We need to add the real YouTube and Meetup links. 😊 

```suggestion
Kevin Baldwyn is a lead engineer at EDF energy and in this talk he will explain how EDF is using EDA to manage the complexity of UK Energy Interactions and describe some of the architecture, patterns, lessons learned, and why you may want to start using EDA in your applications.

You will be able to join us in-person at [Spaces](https://www.spacesworks.com/brighton/trafalgar-place/) or [online](https://www.youtube.com/watch?v=FIXME).

If you're thinking of joining us in-person, please sign up to the event on [meetup.com](https://www.meetup.com/async-web-tech-meetup/events/FIXME/). We also highly recommend you sign up to the [Async Slack](https://join.slack.com/t/asyncjs/shared_invite/zt-1aguxx86q-XjF_yWcFoJ8fyYYzoqgDaQ) for more info, and updates.
```